frontend/src/__tests__/Init.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {render, cleanup} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
you can export it to global test setup using `setupFilesAfterEnv` option in jest